opencti-platform/opencti-graphql/src/domain/stixCyberObservable.js

```diff
@@ -216,6 +215,9 @@ export const addStixCyberObservable = async (context, user, input) => {
     update,
     ...input[graphQLType]
   };
+  if (internal_id) {
```

Comment:
Could it be possible to have a case where `internal_id` from the input is `undefined` or `null` and we want to keep this null value ? Or it is sure that in any case we don't want it ?

Reply:
I don't undesrtand in which case we would have an internal_id in the observable input, it's not part of the schema. And internal_id is not part of the stream, right ? @JeremyCloarec